docs/guide/en/typecasting.md

```diff
@@ -139,6 +142,28 @@ $money = $hydrator->create(Money::class, [
 ]);
 ```
 
+### `Trim` / `LeftTrim` / `RightTrim`
+
+To strip whitespace (or other characters) from the beginning and/or end of a resolved string value, you can use `Trim`,
+`LeftTrim` or `RightTrim` attributes:
```

Comment:
Is it better to name these `TrimLeft`/`TrimRight`?

Reply:
Thanks for your suggestion. I've created separate issue - #91, because it's out of scope of current PR.